packages/foam-core/src/utils.ts

```diff
@@ -1,4 +1,34 @@
+import path from 'path';
+import crypto from 'crypto';
 import { titleCase } from 'title-case';
+import GithubSlugger from 'github-slugger';
+import { URI, ID } from 'types';
+
+export const hash = (text: string) =>
+  crypto
+    .createHash('md5')
```

Comment:
in case we keep using path hashes, might make sense to use sha1: https://medium.com/@chris_72272/what-is-the-fastest-node-js-hashing-algorithm-c15c1a0e164e . Also git uses sha1 IIRC.

Reply:
I am super easy about that, happy to change it unless @jevakallio has any objection

Reply:
Feel free to change, I don't have any issues as these don't need to be secure hashes and I have never experienced collisions with sha.

Keeping in mind, this is in foam-core, so technically we might want to support using it in a browser based environment -- though I think we might have leaked a bunch of other node APIs in here as well. If we wanted to be good about this, could consider murmurhash.